Failed to build on Emscripten on qt 6.8.+

Hello, whilst building quazip for emscripten on qt 6.8.3, the build stops with the following error message:

> quazip/quazip/quazip_qt_compat.h:114:65: error: no member named 'utc' in 'QTimeZone'
        return QDateTime(QDate(1601, 1, 1), QTime(0, 0), QTimeZone::utc());
                                                                                             
This issue is pretty easy to fix, so I will make a Pull Request.

No, the error message indicates that `QTimeZone::utc()`simply does not exist, nothing else. This is fixable by changing that to `QTimeZone::UTC`. I do not know why that happend. 

This is the error message I received when using quazip with emscripten with a github action. 

```[48/80] Building CXX object libraries/quazip/quazip/CMakeFiles/QuaZip.dir/QuaZip_autogen/mocs_compilation.cpp.o
FAILED: [code=1] libraries/quazip/quazip/CMakeFiles/QuaZip.dir/QuaZip_autogen/mocs_compilation.cpp.o 
/home/runner/work/StudyGuide/StudyGuide/emsdk/upstream/emscripten/em++ -DQT_CORE5COMPAT_LIB -DQT_CORE_LIB -DQUAZIP_STATIC -D_LARGEFILE64_SOURCE=1 -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip -I/home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip/QuaZip_autogen/include -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/zlib/libz.a -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/zlib/zconf.h -I/home/runner/work/StudyGuide/StudyGuide/libraries/quazip -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/zlib -I/home/runner/work/StudyGuide/StudyGuide/libraries/zlib -isystem /home/runner/work/StudyGuide/Qt/6.8.3/wasm_multithread/include/QtCore -isystem /home/runner/work/StudyGuide/Qt/6.8.3/wasm_multithread/include -isystem /home/runner/work/StudyGuide/Qt/6.8.3/wasm_multithread/mkspecs/wasm-emscripten -isystem /home/runner/work/StudyGuide/Qt/6.8.3/wasm_multithread/include/QtCore5Compat -g -std=gnu++17 -pthread -MD -MT libraries/quazip/quazip/CMakeFiles/QuaZip.dir/QuaZip_autogen/mocs_compilation.cpp.o -MF libraries/quazip/quazip/CMakeFiles/QuaZip.dir/QuaZip_autogen/mocs_compilation.cpp.o.d -o libraries/quazip/quazip/CMakeFiles/QuaZip.dir/QuaZip_autogen/mocs_compilation.cpp.o -c /home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip/QuaZip_autogen/mocs_compilation.cpp
In file included from /home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip/QuaZip_autogen/mocs_compilation.cpp:4:
In file included from /home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip/QuaZip_autogen/EWIEGA46WW/moc_quazipfile.cpp:9:
In file included from /home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip/QuaZip_autogen/EWIEGA46WW/../../../../../../libraries/quazip/quazip/quazipfile.h:31:
In file included from /home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip/QuaZip_autogen/EWIEGA46WW/../../../../../../libraries/quazip/quazip/quazip.h:30:
/home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip/QuaZip_autogen/EWIEGA46WW/../../../../../../libraries/quazip/quazip/quazip_qt_compat.h:110:65: error: no member named 'utc' in 'QTimeZone'
  110 |     return QDateTime(QDate(1970, 1, 1), QTime(0, 0), QTimeZone::utc());
      |                                                      ~~~~~~~~~~~^
/home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip/QuaZip_autogen/EWIEGA46WW/../../../../../../libraries/quazip/quazip/quazip_qt_compat.h:114:65: error: no member named 'utc' in 'QTimeZone'
  114 |     return QDateTime(QDate(1601, 1, 1), QTime(0, 0), QTimeZone::utc());
      |                                                      ~~~~~~~~~~~^
2 errors generated.
[49/80] Building CXX object libraries/quazip/quazip/CMakeFiles/QuaZip.dir/JlCompress.cpp.o
FAILED: [code=1] libraries/quazip/quazip/CMakeFiles/QuaZip.dir/JlCompress.cpp.o 
/home/runner/work/StudyGuide/StudyGuide/emsdk/upstream/emscripten/em++ -DQT_CORE5COMPAT_LIB -DQT_CORE_LIB -DQUAZIP_STATIC -D_LARGEFILE64_SOURCE=1 -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip -I/home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip/QuaZip_autogen/include -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/zlib/libz.a -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/zlib/zconf.h -I/home/runner/work/StudyGuide/StudyGuide/libraries/quazip -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/zlib -I/home/runner/work/StudyGuide/StudyGuide/libraries/zlib -isystem /home/runner/work/StudyGuide/Qt/6.8.3/wasm_multithread/include/QtCore -isystem /home/runner/work/StudyGuide/Qt/6.8.3/wasm_multithread/include -isystem /home/runner/work/StudyGuide/Qt/6.8.3/wasm_multithread/mkspecs/wasm-emscripten -isystem /home/runner/work/StudyGuide/Qt/6.8.3/wasm_multithread/include/QtCore5Compat -g -std=gnu++17 -pthread -MD -MT libraries/quazip/quazip/CMakeFiles/QuaZip.dir/JlCompress.cpp.o -MF libraries/quazip/quazip/CMakeFiles/QuaZip.dir/JlCompress.cpp.o.d -o libraries/quazip/quazip/CMakeFiles/QuaZip.dir/JlCompress.cpp.o -c /home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip/JlCompress.cpp
In file included from /home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip/JlCompress.cpp:26:
In file included from /home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip/JlCompress.h:29:
In file included from /home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip/quazip.h:30:
/home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip/quazip_qt_compat.h:110:65: error: no member named 'utc' in 'QTimeZone'
  110 |     return QDateTime(QDate(1970, 1, 1), QTime(0, 0), QTimeZone::utc());
      |                                                      ~~~~~~~~~~~^
/home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip/quazip_qt_compat.h:114:65: error: no member named 'utc' in 'QTimeZone'
  114 |     return QDateTime(QDate(1601, 1, 1), QTime(0, 0), QTimeZone::utc());
      |                                                      ~~~~~~~~~~~^
2 errors generated.
[50/80] Building CXX object libraries/quazip/quazip/CMakeFiles/QuaZip.dir/quaziodevice.cpp.o
[51/80] Building CXX object libraries/quazip/quazip/CMakeFiles/QuaZip.dir/quagzipfile.cpp.o
[52/80] Building CXX object libraries/quazip/quazip/CMakeFiles/QuaZip.dir/quazip.cpp.o
FAILED: [code=1] libraries/quazip/quazip/CMakeFiles/QuaZip.dir/quazip.cpp.o 
/home/runner/work/StudyGuide/StudyGuide/emsdk/upstream/emscripten/em++ -DQT_CORE5COMPAT_LIB -DQT_CORE_LIB -DQUAZIP_STATIC -D_LARGEFILE64_SOURCE=1 -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip -I/home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/quazip/quazip/QuaZip_autogen/include -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/zlib/libz.a -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/zlib/zconf.h -I/home/runner/work/StudyGuide/StudyGuide/libraries/quazip -I/home/runner/work/StudyGuide/StudyGuide/build/libraries/zlib -I/home/runner/work/StudyGuide/StudyGuide/libraries/zlib -isystem /home/runner/work/StudyGuide/Qt/6.8.3/wasm_multithread/include/QtCore -isystem /home/runner/work/StudyGuide/Qt/6.8.3/wasm_multithread/include -isystem /home/runner/work/StudyGuide/Qt/6.8.3/wasm_multithread/mkspecs/wasm-emscripten -isystem /home/runner/work/StudyGuide/Qt/6.8.3/wasm_multithread/include/QtCore5Compat -g -std=gnu++17 -pthread -MD -MT libraries/quazip/quazip/CMakeFiles/QuaZip.dir/quazip.cpp.o -MF libraries/quazip/quazip/CMakeFiles/QuaZip.dir/quazip.cpp.o.d -o libraries/quazip/quazip/CMakeFiles/QuaZip.dir/quazip.cpp.o -c /home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip/quazip.cpp
In file included from /home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip/quazip.cpp:29:
In file included from /home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip/quazip.h:30:
/home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip/quazip_qt_compat.h:110:65: error: no member named 'utc' in 'QTimeZone'
  110 |     return QDateTime(QDate(1970, 1, 1), QTime(0, 0), QTimeZone::utc());
      |                                                      ~~~~~~~~~~~^
/home/runner/work/StudyGuide/StudyGuide/libraries/quazip/quazip/quazip_qt_compat.h:114:65: error: no member named 'utc' in 'QTimeZone'
  114 |     return QDateTime(QDate(1601, 1, 1), QTime(0, 0), QTimeZone::utc());
      |                                                      ~~~~~~~~~~~^
2 errors generated.
ninja: build stopped: subcommand failed.```

--- a/quazip/quazip_qt_compat.h
+++ b/quazip/quazip_qt_compat.h
@@ -107,11 +107,11 @@ inline QString quazip_symlink_target(const QFileInfo &fi) {
 #if QT_VERSION >= QT_VERSION_CHECK(6, 8, 0)
 #include <QtCore/QTimeZone>
 inline QDateTime quazip_since_epoch() {
-    return QDateTime(QDate(1970, 1, 1), QTime(0, 0), QTimeZone::utc());
+    return QDateTime(QDate(1970, 1, 1), QTime(0, 0), QTimeZone::UTC);
 }
 
 inline QDateTime quazip_since_epoch_ntfs() {
-    return QDateTime(QDate(1601, 1, 1), QTime(0, 0), QTimeZone::utc());
+    return QDateTime(QDate(1601, 1, 1), QTime(0, 0), QTimeZone::UTC);
 }
 #else
 inline QDateTime quazip_since_epoch() {

